pages/details/[id].js

```diff
@@ -1,33 +1,22 @@
 import EventDetails from "@/components/EventDetails";
 import { useRouter } from "next/router";
 import selectImage from "@/components/utils";
-import Link from "next/link";
 import styled from "styled-components";
 
 const Wrapper = styled.div`
   display: flex;
   flex-direction: column;
-  flex-direction: column;
   align-items: center;
-  width: 100%;
 `;
 
 const PageTitle = styled.h1`
-  text-align: center;
   margin: 30px 0;
+  text-align: center;
 `;
 
-const Nav = styled.nav`
+const Header = styled.div`
```

Comment:
You could use a `[<header>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header?retiredLocale=de)` element here. 
```suggestion
const Header = styled.header`
```